SwiftiumKit/Crypto.m

```diff
@@ -74,7 +75,7 @@ void sk_crypto_hmac_sha1(const void *data, unsigned int dataLength, const void *
     char keyPtr[keySize+1]; // room for terminator (unused)
     bzero(keyPtr, keySize+1); // fill with zeroes (for padding)
     
-    memcpy(keyPtr, key, keySize+1);
+    memcpy(keyPtr, key, MAX(keyLength, keySize));
```

Comment:
Some size refer to `keySize+1` and some to `MAX(keyLength, keySize)`, which one is the good one ?

Reply:
fixed up by rewriting history to remove this change